satellite/metainfo/metainfo.go

```diff
@@ -1262,6 +1262,7 @@ func (endpoint *Endpoint) GetObject(ctx context.Context, req *pb.ObjectGetReques
 			index++
 		}
 	}
+	endpoint.log.Sugar().Info("Authorize get object for project ID ", keyInfo.ProjectID)
```

Comment:
Valid for all changes:

1. Can we use structured logging, i.e.
```go
endpoint.log.Info("Get Object", zap.Stringer("Project ID", keyInfo.ProjectID))
```
2. Can we have the log message right after `keyInfo` variable is retrieved instead of at the end of the method?

3. Can we have the log message for all endpoint methods?

Reply:
1. Sure
2. The idea was to have the log only if there has been no error. If there is, it is already on the log. But I can add it :)
3. Sure. But is it not going to be too much for an Info log? I don't know how it will look like on a real satellite, but I'm afraid it won't be readable.

Reply:
OK, you are right for 2 and 3.